config/samples/databricks_v1alpha1_djob.yaml

```diff
@@ -1,22 +1,23 @@
 apiVersion: databricks.microsoft.com/v1alpha1
```

Comment:
Could you please add second file databricks_v1alpha1_djob_1.yaml  instead of changing databricks_v1alpha1_djob.yaml? It's good to have two examples.

Reply:
good idea

Reply:
Could you please revert back your changes on databricks_v1alpha1_djob.yaml?

Reply:
reverted